src/controller/SetUpCodePairer.cpp

```diff
@@ -483,11 +466,20 @@ void SetUpCodePairer::OnPairingComplete(CHIP_ERROR error)
         ResetDiscoveryState();
         if (pairingDelegate != nullptr)
         {
+            PASEEstablishmentComplete();
```

Comment:
Why do we only do this if `pairingDelegate` is non-null?

Reply:
There is no good reason...